src/Arcus.Templates.Tests.Integration/AzureFunctions/ServiceBus/AzureFunctionsServiceBusProject.cs

```diff
@@ -134,17 +134,14 @@ public class AzureFunctionsServiceBusProject : AzureFunctionsProject, IAsyncDisp
 
         private void AddOrderMessageHandlerImplementation()
         {
-            AddPackage("Arcus.EventGrid.Core", "3.3.0");
-
             AddTypeAsFile<Order>();
             AddTypeAsFile<Customer>();
-            AddTypeAsFile<OrderCreatedEvent>();
             AddTypeAsFile<OrderCreatedEventData>();
 
-            AddTypeAsFile<TestOrdersAzureServiceBusMessageHandler>();
+            AddTypeAsFile<WriteToFileMessageHandler>();
```

Comment:
Used a `WriteToFileMessageHandler` here. Tt actually makes the integration tests a bit more stable, as we do not need to interact with EventGrid anymore. We inject a message handler that writes the message on disk instead and look for such a file during the assertion. It also removes the need to deploy such an EventGrid resource for the Templates repo + it was actually unnecessary additional overhead to bring EventGrid in the loop here, as we only need to test if the generated project is capable of routing messages.

Reply:
Indeed, if we only need to test the routing, no need to add additional complexity and dependencies.  I like it simple :)